bench/Sender/Main.hs

```diff
@@ -56,7 +57,7 @@ main = do
     loadLogConfig logsPrefix logConfig
     setLocaleEncoding utf8
 
-    Right transport_ <- TCP.createTransport "0.0.0.0" "3432" TCP.defaultTCPParameters
+    Right transport_ <- TCP.createTransport "0.0.0.0" "127.0.0.1" "3432" TCP.defaultTCPParameters
```

Comment:
Perhaps it would be best to leave nt-tcp unpatched and instead take the bind address of the node as a config parameter. Binding on `0.0.0.0` will not be allowed, and you'll have to know your public address.